tests/integration_tests/event/test_delete.py

```diff
@@ -6,7 +6,20 @@
 from tests.integration_tests.setup_db import setup_db
 
 
-def test_delete_event(client: TestClient, data_db: Session, user_header_token):
+def test_delete_event_super(
+    client: TestClient, data_db: Session, superuser_header_token
+):
+    setup_db(data_db)
+    response = client.delete("/api/v1/events/E.0.0.2", headers=superuser_header_token)
+    assert response.status_code == status.HTTP_200_OK
+    assert data_db.query(FamilyEvent).count() == 2
+    assert (
+        data_db.query(FamilyEvent).filter(FamilyEvent.import_id == "E.0.0.2").count()
```

Comment:
Is there an endpoint or something we can call as well to validate that the system responds as expected given a not found event as well?